starter-code/src/app/component/log-form-component/log-form-component.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, OnInit } from '@angular/core';
+import { PeopleService } from '../../services/people.service'
+
+
+@Component({
+  selector: 'app-log-form-component',
+  templateUrl: './log-form-component.component.html',
+  styleUrls: ['./log-form-component.component.css']
+})
+export class LogFormComponentComponent implements OnInit {
+
+  peopleService: PeopleService;
+  showMsg: boolean = false;
+  feedbackEnable: boolean = false;
+  proccessing: boolean = false;
+
+  constructor(peopleService: PeopleService) {
+    this.peopleService = peopleService;
+  }
+
+  ngOnInit() {
+  }
+
+  showMessage(){
+    this.showMsg = true;
+  }
+
+  addAccessItem(form) {
```

Comment:
This method IS NOT THE SAME that you have in your service. So, if the form is valid, you need to call the method:
```
this.peopleService.addAccessItem(this.name, this.message);
```
But be careful, because you are two-way binding name and message in your form and not declaring them as properties of the component.
Last, but not least, maybe is not the best idea put the same name for the method in your component and  your service.